### ISSUE TITLE
Add padding to the image and modify standard messages for the errors field

### ISSUE THREAD

**Issue by Azzeccagarbugli** [NearHuscarl/flutter_login#6]
I'm working with this **awesome** plugin since few days and I really enjoy it but I found out some issues that I don't know how to fix.

When the field **are empty** there are these error messages that unfortunately I'm not able to change or translate even if there is the class `LoginMessages` that somehow already implement this mechanism: 

![Screenshot_20191114_194936](https://user-images.githubusercontent.com/16477661/68886791-1fa8ff00-0718-11ea-8fd3-52210317cae4.png)

The second question that I would like to ask is: is there any possibility to add a `Padding` to the image above the title of the widget so that is not to close to the top of the screen?

![Screenshot_20191114_195447](https://user-images.githubusercontent.com/16477661/68887098-a52caf00-0718-11ea-8490-0efb32c55277.png)

Thanks a lot in advance!

**Comment by DocSnibble** [comment]
I found a solution to your first problem: 

The error messages for the input fields are generated in the respective validator.
So for the Email field put the following code inside FlutterLogin() :

```
emailValidator: (value) {
    if (value.isEmpty) {
        return 'E-Mail cannot be empty';
    }
    return null;
},
```

**Comment by juliansteenbakker** [comment]
Fixed in #132
